test/test_example.py

```diff
@@ -89,13 +89,28 @@ def test_flaky_function(param=[]):
     assert param_length == 1
 
 
+@flaky
 @genty
-class ExampleFlakyTestsWithUnicodeTestNames(ExampleFlakyTests):
-    @genty_dataset('ascii name', 'ńőń ȁŝćȉȉ ŝƭȕƒƒ')
-    def test_non_flaky_thing(self, message):
+class ExampleFlakyTestsWithTrickyTestNames(TestCase):
+    _threshold = -1
+
+    _unicode_string = 'Plain Hello'
+    _byte_string = str(_unicode_string)
```

Comment:
Needs to be `binary_type`, not `str`.

If we remove the `six` dependency, maybe move the definitions of `binary_type` and `unicode_type` into a util module, so they can also be accessed here.